pkg/api/v1/types.go

```diff
@@ -1122,6 +1122,10 @@ type AzureFileVolumeSource struct {
 	// the ReadOnly setting in VolumeMounts.
 	// +optional
 	ReadOnly bool `json:"readOnly,omitempty" protobuf:"varint,3,opt,name=readOnly"`
+	// the namespace of the secret that contains Azure Storage Account Name and Key
+	// default is the same as the Pod
+	// +optional
+	SecretNameSpace *string `json:"secretNamespace" protobuf:"bytes,4,opt,name=secretNamespace"`
```

Comment:
cc @kubernetes/sig-storage-api-reviews on how you want to handle this... this is the same issue as the storageos local (from a pod) or fully-qualified (from a pv) secret reference. storageos ended up with two volume source types, but the azurefilevolumesource type already exists.